tests/bluetooth/crypto_toolbox/Kconfig

```diff
@@ -0,0 +1,9 @@
+config BT_CRYPTO_TOOLBOX_TEST
+    bool
+    default y
+    select BT_CRYPTO_TOOLBOX
+    help
```

Comment:
don't need a helptext since it's just a shim/hack to select the non-user-facing `BT_CRYPTO_TOOLBOX`.
I would name it `BT_ENABLE_CRYPTO_TOOLBOX` so it's clear what it does.